Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -132,23 +141,43 @@ class Connection : public ThreadSafeRefCounted<Connection, WTF::DestructionThrea
     };
 #endif
 
+#if USE(UNIX_DOMAIN_SOCKETS)
+    using Handle = UnixFileDescriptor;
+#elif OS(WINDOWS)
+    using Handle = Win32Handle;
+#elif OS(DARWIN)
+    using Handle = MachSendRight;
+#endif
```

Comment:
You removed SharedMemory::IPCHandle. https://bugs.webkit.org/show_bug.cgi?id=240056
Connection::Handle is like another IPCHandle. Can you remove it?

Reply:
No.
SharedMemory needs handle (SharedMemory::Handle) to serialise SharedMemory reference to a handle.
Connection needs a handle (Connection::Handle) to serialise Connection to a handle.

Reply:
Does Connection need both Handle and Identifier?

Reply:
Currently yes. #4059 is working towards removing Connection::Identifier